control-plane-operator/controllers/hostedcontrolplane/pki/etcd.go

```diff
@@ -42,8 +41,8 @@ func ReconcileEtcdServerSecret(secret, ca *corev1.Secret, ownerRef config.OwnerR
 
 func ReconcileEtcdPeerSecret(secret, ca *corev1.Secret, ownerRef config.OwnerRef) error {
 	dnsNames := []string{
-		fmt.Sprintf("*.etcd.%s.svc", secret.Namespace),
-		fmt.Sprintf("*.etcd.%s.svc.cluster.local", secret.Namespace),
+		fmt.Sprintf("*.etcd-discovery.%s.svc", secret.Namespace),
```

Comment:
Why rename this, this is used for peer communication and not only discovery or not?